README.md

```diff
@@ -1456,14 +1467,14 @@ To put metric data.   [UnitTypes](http://docs.aws.amazon.com/AmazonCloudWatch/la
 (subscribe :protocol "email"
            :topic-arn "arn:aws:sns:us-east-1:676820690883:my-topic"
            :endpoint "mcohen01@gmail.com")
-           
+
 (subscribe :protocol "lambda"
            :topic-arn "arn:aws:sns:us-east-1:676820690883:my-topic"
            :endpoint "arn:aws:lambda:us-east-1:676820690883:function:my-function")
 
 ;; provide endpoint in creds for topics in non-default region
 (subscribe {:endpoint "eu-west-1"}
-	   :protocol "lambda"
+       :protocol "lambda"
```

Comment:
This looks a little messed up, but perhaps I should just run cljfmt in a different PR.